src/wrappers/themis/wasm/src/index.ts

```diff
@@ -166,3 +165,47 @@ class InitializedPromise {
 export const initialized = new InitializedPromise((resolve) => {
   onRuntimeInitialized = resolve;
 });
+
+// Historically, WasmThemis allowed to use default import in ES6 code:
+//
+//     import themis from 'wasm-themis';
+//
+// Make sure it still works.
+
+export default {
+  initialize,
+  initialized,
+  SecureCellSeal,
+  SecureCellTokenProtect,
+  SecureCellContextImprint,
+  SecureMessage,
+  SecureMessageSign,
+  SecureMessageVerify,
+  SecureSession,
+  SecureComparator,
+  ThemisError,
+  ThemisErrorCode,
+  SymmetricKey,
+  KeyPair,
+  PrivateKey,
+  PublicKey,
+};
+
+// And of course export all public things individually.
+
+export {
```

Comment:
wat?) I need to go on some nodejs/es6 courses and learn how works new imports in javascript world....

Reply:
It's not like I have much clue myself, lol, I'm just reading the MDN:

- [JavaScript Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
- [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)
- [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)